src/Microsoft.DotNet.Interactive.HttpRequest.Tests/ParserTests.Comments.cs

```diff
@@ -22,11 +23,25 @@ public void line_comment_before_method_and_url_is_parsed_correctly()
             var result = Parse(code);
 
             var methodNode = result.SyntaxTree.RootNode
```

Comment:
Rename to `requestNode`

Reply:
Or perhaps avoid the variable entirely and combine the two assertion statements into one.